enterprise/utils.py

```diff
@@ -399,6 +398,56 @@ def get_enterprise_customer_for_user(auth_user):
     return enterprise_customer_user.enterprise_customer
 
 
+def get_enterprise_customer_user(user_id, enterprise_uuid):
+    """
+    Return the object for EnterpriseCustomerUser.
+
+    Arguments:
+        user_id (str): user identifier
+        enterprise_uuid (UUID): Universally unique identifier for the enterprise customer.
+
+    Returns:
+        (EnterpriseCustomerUser): enterprise customer user record
+    """
+    try:
+        # pylint: disable=no-member
+        enterprise_customer_user = enterprise.models.EnterpriseCustomerUser.objects.get(
+            enterprise_customer__uuid=enterprise_uuid,
+            user_id=user_id
+        )
+    except enterprise.models.EnterpriseCustomerUser.DoesNotExist:
```

Comment:
@haikuginger we are doing this because there is circular import in `models.py` from `utils.py` and `api.py`.